### PR TITLE
Update for v2 api

### PR DESCRIPTION
**Pull request by iantance**
@stevenmiller888 Here's the update for the Drip integration. Let me know if there's anything else that needs to change.

### CODE REVIEW

**Comment by stevenmiller888** [comment]
@amillet89 @ndhoule @f2prateek Can you review Ian's PR? We'll also need to expose the API token in our interface for them.

**Comment by iantance** [comment]
@stevenmiller888 Any word on this?

File: lib/index.js [hunk 1]
```diff
@@ -45,7 +43,15 @@ Drip.prototype.identify = function(identify, fn){
     .type('json')
     .send({ subscribers: [subscriber] })
     .end(function(err, res){
-      if (err) return fn(err);
+      if (err) {
```
Comment (f2prateek) [inline lib/index.js]: not sure if we need to do this grunt work, the `err` variable should capture the request and whatever was returned.

Comment (iantance) [inline lib/index.js]: I've removed all that. Removed the original  "if (err) return fn(err);" and tests pass.

File: lib/mapper.js [hunk 2]
```diff
@@ -30,7 +26,8 @@ exports.identify = function(msg){
 
 exports.track = function(msg){
   var ret = {};
-  if (msg.revenue()) ret.value = msg.revenue();
+  ret.properties = {};
+  if (msg.revenue()) ret.properties.value = Math.round(msg.revenue() * 100);
```
Comment (f2prateek) [inline lib/mapper.js:30]: Does drip only accept revenue in cents?

File: lib/index.js [hunk 3]
```diff
@@ -78,18 +75,6 @@ Drip.prototype.track = function(track, fn){
     .end(this.handle(fn));
 };
 
-/**
- * Get campaignId from options or settings.
```
Comment (f2prateek) [inline lib/index.js:82]: So campaignId is not required anymore?

File: lib/mapper.js (hunk 2 above)
Comment (iantance) [inline lib/mapper.js:30]: Our internal conversion tracking expects the "value" on events to be sent in cents.

File: lib/index.js (hunk 3 above)
Comment (iantance) [inline lib/index.js:82]: Correct. Drip subscribers are not necessarily connected with a campaign, and our current API reflects that.

**Comment by f2prateek** [comment]
Sweet, LGTM!

**Comment by f2prateek** [comment]
@iantance forgot to ask, but would we need to update our analytics.js integration as well? Might confuse users if the server side and client integrations work differently.

**Comment by iantance** [comment]
I don't think an update to analytics.js should be necessary. The v2 upgrade
only applied our REST API, and it's only the server side integration that
isn't working currently. Analytics.js pushes calls to our JS API. The
client side version is being used successfully by a number of our customers.

**Comment by f2prateek** [comment]
Cool, just checking. Thanks for the PR!

**Pull request closed without merging**